inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -98,12 +98,83 @@ def article_exists(obj, eng):
     """
     matches = dedupe_list(match(obj.data))
     record_ids = [el['_source']['control_number'] for el in matches]
-    if record_ids:
-        obj.extra_data['record_matches'] = record_ids
-        return True
+    obj.extra_data.setdefault('matches', {})['exact'] = record_ids
+    return bool(record_ids)
 
-    obj.extra_data['record_matches'] = []
-    return False
+
+@with_debug_logging
+def fuzzy_match(obj, eng):
+    """Return ``True`` if a similar record is found in the system.
+
+    Uses a custom configuration for ``inspire-matcher`` to find records
+    similar to the current workflow object's payload in the system.
+
+    Also sets the ``matches.fuzzy`` property in ``extra_data`` to the list of
+    control numbers that matched.
+
+    Arguments:
+        obj: a workflow object.
+        eng: a workflow engine.
+
+    Returns:
+        bool: ``True`` if the workflow object has a duplicate in the system
+        ``False`` otherwise.
+
+    """
+    fuzzy_match_configuration = {
+        'algorithm': [
+            {
+                'queries': [
+                    {
+                        'clauses': [
+                            {
+                                'boost': 20,
+                                'path': 'abstracts',
+                            },
+                            {
+                                'boost': 10,
+                                'path': 'authors[:3]',
+                            },
+                            {
+                                'boost': 20,
+                                'path': 'titles',
+                            },
+                            {
+                                'boost': 10,
+                                'path': 'report_numbers',
+                            },
+                        ],
+                        'type': 'fuzzy',
+                    }
+                ]
+            }
+        ],
+        'doc_type': 'hep',
+        'index': 'records-hep',
+    }
```

Comment:
This should be part of the configuration, so that it can be overridden in the OPS repo (if need arises).

Reply:
Moving it in `inspirehep/config.py`.

Reply:
Something happened here... wasn't this config moved?